src/main/java/org/folio/innreach/domain/listener/KafkaInventoryEventListener.java

```diff
@@ -146,4 +142,10 @@ else if (event.getType().equals(DELETED)) {
       }
     }
   }
+
+  private List<UUID> getCentralServerIds() {
+    Page<UUID> ids = centralServerRepository.getIds(new OffsetRequest(0, 2000));
```

Comment:
Do we want to make this value 2000 configurable ? Or this is an optimum limit to fetch the records ?

Reply:
I think this is fine. our need is to fetch all records of centralserverIds so that we can add a event against each centralServerid in database

Reply:
@Vignesh-kalyanasundaram  can you say some more about why it shouldn't be configurable? Not sure I understand your comment.

Reply:
I don't mean that it shouldn't be configurable. I mean to say that we need to fetch all the centralServerIds before saving the event into DB. As the existing code implemented in this manner, ([please see here](https://github.com/folio-org/mod-inn-reach/blob/master/src/main/java/org/folio/innreach/domain/service/impl/ContributionActionServiceImpl.java#L255)), I followed the same